kibble/setup/kibble.ini

```
@@ -0,0 +1,18 @@
+[accounts]
+allowSignup = True
+verify = True
+
+[api]
+database = 2
+version = 0.1.0
+
+[elasticsearch]
+dbname = kibble
+host = elasticsearch
+port = 9200
```

```suggestion
conn_uri = elasticsearch:9200
```
Let's consolidate it as mentioned in #52